@here/harp-webtile-datasource/lib/WebTileDataSource.ts

```diff
@@ -96,12 +100,38 @@ export class WebTileDataSource extends DataSource {
 
         this.dataProvider = this.m_options.dataProvider;
         this.cacheable = true;
+        this.m_opacity = this.m_options.renderingOptions?.opacity ?? 1;
+        this.m_transparent =
+            this.m_options.renderingOptions?.transparent === true || this.m_opacity < 1;
+        this.m_renderOrder = this.m_options.renderingOptions?.renderOrder ?? 0;
+
         this.m_resolution = getOptionValue(
             m_options.resolution,
             WebTileDataSource.resolutionValue.resolution512
         );
     }
 
+    /**
+     * Sets the opacity for the WebTileDataSource, will only affect not yet loaded or not cached
+     * tiles.
+     *
+     * Use WebTileDataSource:clearCache and MapView:markTilesDirty to reload all tiles with the
+     * new opacity setting.
+     */
+    set opacity(value: number) {
+        this.m_opacity = value;
+        if (this.m_opacity < 1) {
+            this.m_transparent = true;
```

Comment:
In the construtor m_transparent is set to false if `opacity === 1` I guess you want to do the same here.

Reply:
No, I dont want to override it, in case it was set in the renderOptions, i.e. to force the material to handle transparencies in the texture

Reply:
but you override it. If transparent is false in the options and you set opacity to 0.9 and later back to 1.0 it will stay true even though it is false in the options.

Reply:
yes, thats true, but it will visually not really make a difference. Thats why I kept it on the same variable. 
But still, you are right, I ll add a condition that solves both cases